Libs/MRML/Widgets/qMRMLLabelComboBox.cxx

```diff
@@ -126,20 +126,6 @@ qMRMLLabelComboBox::qMRMLLabelComboBox(QWidget* newParent)
 // --------------------------------------------------------------------------
 qMRMLLabelComboBox::~qMRMLLabelComboBox() = default;
 
-// ----------------------------------------------------------------
-void qMRMLLabelComboBox::printAdditionalInfo()
```

Comment:
I don't know about any particular use of this method, but was there any specific motivation to remove it?

Reply:
It addresses a warning about deprecated use of  'endl'. See details in commit message.

I can also revisit and instead address the warning.

Reply:
In general, we should just change the API for a good reason. Since it seems easy to update and maintain this method and it may be useful as a kind of a "PrintSelf", I think we should keep it.

Reply:
> it may be useful as a kind of a "PrintSelf"

Indeed, that was the original motivation when we started to develop the widgets ..

In the meantime, I will fix the warnings and keep the method.

Reply:
This will be "properly" fixed after https://github.com/commontk/CTK/pull/1008 is integrated.